library/rssbuilder/rss_sample_script.php

```diff
@@ -21,7 +21,7 @@
 ob_start();
 include_once 'class.RSSBuilder.inc.php';
 
-/* create the object - remember, not all attibutes are supported by every rss version. just hand over an empty string if you don't need a specific attribute */
+/* create the object - remember, not all attributes are supported by every rss version. just hand over an empty string if you don't need a specific attribute */
```

Comment:
Remove this change, there is no point in patching comments in a 3rd party library